src/backend/distributed/transaction/multi_shard_transaction.c

```diff
@@ -92,6 +92,7 @@ CreateShardConnectionHash(void)
 	info.keysize = sizeof(int64);
 	info.entrysize = sizeof(ShardConnections);
 	info.hash = tag_hash;
+	info.hcxt = CurrentMemoryContext;
```

Comment:
Nope. So if I switch that is this a ship it?

--
*Jason Petersen*
Software Engineer | Citus Data
303.736.9255
jason@citusdata.com

On Sep 2, 2016, at 4:33 PM, Andres Freund <notifications@github.com> wrote:

I'd be more comfortable with using TopTransactionContext. Do you see any
arguments against that?